addons/mass_mailing/static/src/scss/mass_mailing.wysiwyg.scss

```diff
@@ -42,6 +43,7 @@
         }
     }
     &.o_loaded {
+        display: inline;
```

Comment:
Why `inline` though? From what I could see on runbot the initial display of the sidebar before your `display:none` fix is not `inline`.

Reply:
Yes, my bad it should be `flex` as it was before , I fixed it